apps/crn-frontend/src/network/working-groups/Outputs.tsx

```diff
@@ -0,0 +1,81 @@
+import { SearchFrame } from '@asap-hub/frontend-utils';
+import { ProfileOutputs } from '@asap-hub/react-components';
+import { useCurrentUserCRN } from '@asap-hub/react-context';
+import { network } from '@asap-hub/routing';
+import { ComponentProps } from 'react';
+
+import { usePagination, usePaginationParams } from '../../hooks';
+import { useWorkingGroupById } from './state';
+
+type OutputsListProps = Pick<
+  ComponentProps<typeof ProfileOutputs>,
+  'hasOutputs' | 'userAssociationMember'
+> & {
+  displayName: string;
+  searchQuery: string;
+  filters: Set<string>;
+  workingGroupId: string;
+};
+type OutputsProps = {
+  workingGroupId: string;
+};
+
+const OutputsList: React.FC<OutputsListProps> = ({
+  workingGroupId,
+  hasOutputs,
+  userAssociationMember,
+}) => {
+  const { currentPage, isListView, cardViewParams, listViewParams } =
+    usePaginationParams();
+  const { numberOfPages, renderPageHref } = usePagination(0, 10);
+  return (
+    <ProfileOutputs
+      researchOutputs={[]}
+      numberOfItems={0}
+      numberOfPages={numberOfPages}
+      currentPageIndex={currentPage}
+      renderPageHref={renderPageHref}
+      isListView={isListView}
+      cardViewHref={
+        network({})
+          .workingGroups({})
+          .workingGroup({ workingGroupId })
+          .outputs({}).$ + cardViewParams
+      }
+      listViewHref={
+        network({})
+          .workingGroups({})
+          .workingGroup({ workingGroupId })
+          .outputs({}).$ + listViewParams
+      }
+      hasOutputs={hasOutputs}
+      userAssociationMember={userAssociationMember}
+      publishingEntity={'Working Group'}
+    />
+  );
+};
+
+const Outputs: React.FC<OutputsProps> = ({ workingGroupId }) => {
+  const hasOutputs = false;
+  const workingGroup = useWorkingGroupById(workingGroupId);
+
+  const currentUserId = useCurrentUserCRN()?.id;
+  const userAssociationMember = !!useWorkingGroupById(
+    workingGroupId,
+  )?.members.filter((member) => member.user.id === currentUserId).length;
+  return (
+    <article>
+      <SearchFrame title="">
+        <OutputsList
+          workingGroupId={workingGroupId}
+          searchQuery={''}
+          filters={new Set()}
+          hasOutputs={hasOutputs}
+          displayName={workingGroup?.title ?? ''}
```

Comment:
Doesn't there have to be a working group? If there isn't shouldn't this page not be found

Reply:
Followed the same approach from team/Outputs.tsx, but I guess you are right, I will render the NotFoundPage in case we cannot get the workingGroup by Id